test/reductions/grid_search/test_grid_search_classification.py

```diff
@@ -22,15 +22,15 @@ def _simple_threshold_data(number_a0, number_a1,
     a1_scores = np.linspace(0, 1, number_a1)
     score_feature = np.concatenate((a0_scores, a1_scores), axis=None)
 
-    A = np.concatenate((a0s, a1s), axis=None)
+    A = pd.DataFrame({"a_col": np.concatenate((a0s, a1s), axis=None)})
```

Comment:
do we want to standardize what we call these columns? In PP I've set up a constants file and keep referring to the column name constants from there. I think it's a good pattern if we ever change the name. Whether we want to share these names is a different question, of course. Maybe not...

Reply:
You mean standardise the names we use in test?